fishy/engine/semifisher/engine.py

```diff
@@ -76,6 +78,11 @@ def toggle_start(self):
         if self.start:
             self.thread = Thread(target=self.run)
             self.thread.start()
+            playsound(helper.manifest_file("beep.wav"), False)
+        else:
+            playsound(helper.manifest_file("beep.wav"), False)
+            time.sleep(0.3)
+            playsound(helper.manifest_file("beep.wav"), False)
```

Comment:
this might make the GUI a bit unresponsive, you can create a new thread to play sound and time.sleep() so that it doesn't block the UI thread... second parameter of playsound makes blocking/nonblocking u can even use that to make it beep twice without even using time.sleep